cli/command/container/completion.go

```diff
@@ -132,6 +132,7 @@ func addCompletions(cmd *cobra.Command, dockerCli command.Cli) {
 	_ = cmd.RegisterFlagCompletionFunc("ulimit", completeUlimit)
 	_ = cmd.RegisterFlagCompletionFunc("user", completion.NoComplete)
 	_ = cmd.RegisterFlagCompletionFunc("userns", completion.FromList("host"))
+	_ = cmd.RegisterFlagCompletionFunc("uts", completion.NoComplete)
```

Comment:
This one _could_ have completion; theres `UTSMode` "host"; https://github.com/docker/cli/blob/32ff200fe6d843843718709ffa94023eb7356108/vendor/github.com/docker/docker/api/types/container/hostconfig.go#L210-L211


(also note to self that we don't provide an enum for that one)